.github/workflows/publish-helm.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
 
     - name: Update helm index
       env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GH_TOKEN: ${{ secrets.token }}
```

Comment:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication

Reply:
Documentation says is `GITHUB_TOKEN` I agree with the identing bit of this PR

Reply:
OK. I was getting confused by the error in:
* https://github.com/NVIDIA/k8s-device-plugin/actions/runs/10300849949

Which was already addressed by the file as shown. Meaning I have to now address:
* https://github.com/NVIDIA/k8s-device-plugin/actions/runs/11615138763
* https://github.com/NVIDIA/k8s-device-plugin/actions/runs/11615080421

Reply:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/.github/workflows/gh-pages.yml#L36